.github/workflows/workflow.yml

```diff
@@ -0,0 +1,81 @@
+name: CI/CD
```

Comment:
As we should update the name of the workflow file we should also be a bot more descriptive with the name, again like `name: Lint, build & push kubernetes-dns-node-cache container image`.

Additionally I would consider also using the `run-name` field, which we could customize a bit more, atm as I understand it it just shows the git message? Maybe that's also ok but I think smth like `run-name: build & push to {{tag}} by {{author}} bec of {{commit message}}` or so...